fontir/src/orchestration.rs

```diff
@@ -0,0 +1,108 @@
+// Helps coordinate the graph execution
```

Comment:
This line looks like it could pass as documentation for the module, in which case use `//!` :)

Reply:
Done